azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -213,6 +213,7 @@ public Project getProject(final String name) {
       fetchedProject = this.projectsByName.get(name);
     } else {
       try {
+        logger.info("Project " + name + " doesn't exist in cache, fetching from DB now.");
```

Comment:
should we make level debug? Also just FYI, with debug level, we can add more detailed logging for better debuggability without concerning overwhelming logging message

Reply:
Currently only logging with level INFO or above will be printed out in our server. We have to change log4j configuration and do a server deployment to enable debug level logging. It might worth considering using log4j2 to dynamically switch logging levels in the future.

Reply:
For new logging code, I would suggest 
"slf4j provides a more efficient API called parameterized logging.
Try to use that format.
see
https://stackoverflow.com/questions/30975026/log4j-implicit-string-formatting

"
See my comment in #1604

Reply:
Agree. Thanks for the suggestion!